hpo_train.py

```diff
@@ -0,0 +1,24 @@
+"""Hyperparameter optimization of train.py for cifar10_cnn"""
+from crayai import hpo
+
+# Set up evaluator
+evaluator = hpo.Evaluator('python ./train.py configs/hpo_cifar10_cnn.yaml --hpo',
+                          nodes=4,
```

Comment:
What does this arg do when the script is run within a slurm job? Is it just ignored?